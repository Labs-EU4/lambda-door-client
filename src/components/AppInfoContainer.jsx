/* eslint-disable jsx-a11y/no-static-element-interactions */
/* eslint-disable jsx-a11y/anchor-is-valid */
/* eslint-disable jsx-a11y/click-events-have-key-events */
import React from 'react';
import { Icon } from 'antd';
import styled from 'styled-components';
import {
  tabletPortrait,
  mobilePortrait,
  mobileLandscape,
  tabletPortraitLarge,
} from '../styles/theme.styles';

function AppInfoContainer({ infoVisible, hideInfo }) {
  return (
    <StyledDiv infoVisible={infoVisible}>
      <TextBox>
        <div className="heading">
          <h2>Features</h2>
        </div>
        <InfoBox>
          <p className="left-outer">
            <Icon type="slack-circle" theme="filled" className="icon" />
            No need to sign up: Lambda students have automatic access. Just sign
            in via Slack.
          </p>
          <p className="left-center">
            <Icon type="edit" theme="filled" className="icon" />
            Leave company reviews, salary reviews and interview reviews,
            publicly or anonymously.
          </p>
          <p className="right-center">
            <Icon type="info-circle" theme="filled" className="icon" />
            Access up to date information about companies in your area or around
            the world.
          </p>
          <p className="right-outer">
            <Icon type="mail" theme="filled" className="icon" />
            Email referrals: ask questions at the click of a button about the
            companies you're interested in.
          </p>
        </InfoBox>
        <a className="back-link" onClick={hideInfo}>
          Go back &nbsp;
          <Icon
            type="up-circle"
            theme="filled"
            style={{ fontSize: 22, color: '#bb1333' }}
            onClick={hideInfo}
          />
        </a>
      </TextBox>
    </StyledDiv>
  );
}

export default AppInfoContainer;

const StyledDiv = styled.div`
  position: absolute;
  background-color: #bb1333;
  transition: all 0.2s ease-in-out;
  top: ${props => (props.infoVisible ? `0%` : `100%`)};
  height: 100vh;
  width: 100vw;
  display: flex;
  align-items: center;
  justify-content: center;
  @media ${tabletPortrait} {
    background: #ffffff;
  }
  @media ${mobileLandscape} {
    background: #ffffff;
    height: 850px;
    top: ${props => (props.infoVisible ? `0%` : `-850px`)};
  }
  @media ${mobilePortrait} {
    background: #ffffff;
    height: 1000px;
    top: ${props => (props.infoVisible ? `0%` : `-1000px`)};
    width: 100%;
  }
`;

const TextBox = styled.div`
  width: 80%;
  height: 600px;
  background-color: white;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  padding: 4rem;
  @media only screen and (max-width: 1270px) {
    padding: 1rem;
  }
  @media ${mobileLandscape} {
    height: 700px;
  }
  @media ${tabletPortraitLarge} {
    height: 1000px;
    padding: 1rem;
  }

  h2 {
    font-size: 35px;
    font-family: 'Roboto';
    margin-bottom: 0;
    text-align: center;
    @media ${mobilePortrait} {
      padding-top: 50px;
      padding-bottom: 30px;
    }
  }
  .back-link {
    width: 100%;
    display: flex;
    justify-content: flex-end;
    align-items: flex-start;
    font-size: 20px;
    margin-top: 30px;
    @media ${mobileLandscape} {
      margin-top: 0;
    }
    @media ${mobilePortrait} {
      margin-top: -20px;
      justify-content: center;
    }
  }
  @media ${mobilePortrait} {
    width: 98%;
    padding: 0;
    height: 1200px;
  }
`;

const InfoBox = styled.div`
  display: flex;
  align-items: center;
  padding: 2rem;
  @media only screen and (max-width: 1270px) {
    padding: 0;
  }
  @media ${tabletPortraitLarge} {
    flex-wrap: wrap;
    height: 800px;
    width: 90%;
    padding: 2rem;
  }
  @media ${tabletPortrait} {
    padding: 0;
    width: 110%;
    flex-wrap: wrap;
  }
  @media ${mobileLandscape} {
    height: 800px;
    flex-wrap: wrap;
  }
  @media ${mobilePortrait} {
    padding: 0;
    width: 100%;
    flex-direction: column;
  }

  p {
    width: 25%;
    height: 200px;
    padding: 2rem;
    font-size: 18px;
    display: flex;
    flex-direction: column;
    align-items: center;
    text-align: center;
    justify-content: space-between;
    margin-top: 100px;
    @media only screen and (max-width: 1270px) {
      padding: 1rem;
    }

    @media ${tabletPortraitLarge} {
      width: 50%;
      padding: 2rem;
    }
    @media ${tabletPortrait} {
      width: 50%;
      height: 170px;
      padding: 0.5rem;
    }
    @media ${mobilePortrait} {
      width: 100%;
      margin-top: 0;
      height: 180px;
    }
    @media ${mobileLandscape} {
      width: 65%;
      height: 155px;
      margin-top: 5px;
    }
  }

  @media ${tabletPortrait} {
    flex-wrap: wrap;
  }

  .left-outer {
    border-right: 1px solid grey;
    .icon {
      font-size: 30px !important;
    }
    @media ${mobilePortrait} {
      border-right: none;
    }
  }

  .left-center {
    border-right: 1px solid grey;
    .icon {
      font-size: 30px !important;
      @media ${mobileLandscape} {
        padding-bottom: 1.8rem;
      }
    }
    @media ${tabletPortraitLarge} {
      border-right: none;
    }
    @media ${tabletPortrait} {
      border-right: none;
    }
    @media ${mobileLandscape} {
      border-right: none;
    }
    @media ${mobilePortrait} {
      border-right: none;
    }
  }

  .right-center {
    border-right: 1px solid grey;
    .icon {
      font-size: 30px !important;
    }
    @media ${mobilePortrait} {
      border-right: none;
    }
  }

  .right-outer {
    .icon {
      font-size: 30px !important;
      @media only screen and (max-width: 1270px) {
        padding-bottom: 2rem;
      }
      @media ${mobileLandscape} {
        padding-bottom: 0;
      }
      @media ${tabletPortraitLarge} {
        padding-bottom: 0.4rem;
      }
      @media ${tabletPortrait} {
        padding-bottom: 2.5rem;
      }
    }
  }
`;
