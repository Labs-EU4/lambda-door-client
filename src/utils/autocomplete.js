/* eslint-disable react/jsx-no-undef */
import React, { useState } from 'react';
import { Form, Input, AutoComplete, Button } from 'antd';
import AddCompanyModal from '../components/AddReview/AddCompanyModal';

const { Option } = AutoComplete;

const renderOption = Item => {
  return (
    // eslint-disable-next-line react/jsx-filename-extension
    <Option key={Item.name} text={Item.name}>
      <p>{Item.name}</p>
    </Option>
  );
};

const AutoCompleted = ({ label, dataSource, placeholder, onChange }) => {
  const [addingCompany, setAddingCompany] = useState(false);
  const [value, setValue] = useState('');

  const options = dataSource.map(renderOption).concat([
    <Option key="all" value="" className="show-all">
      <Button type="button" onClick={() => setAddingCompany(true)}>
        Add Company
      </Button>
    </Option>,
  ]);

  return (
    <>
      <AddCompanyModal
        visible={addingCompany}
        setAddingCompany={setAddingCompany}
        value={value}
      />
      <Form.Item label={label}>
        <AutoComplete
          className="certain-category-search"
          dropdownClassName="certain-category-search-dropdown"
          dropdownMatchSelectWidth={false}
          dropdownStyle={{ width: 300 }}
          size="default"
          style={{ width: '100%' }}
          dataSource={options}
          optionLabelProp="value"
          filterOption={(inputValue, option) => {
            if (
              option.key.toLowerCase().includes(inputValue.toLowerCase()) ||
              option.key.toLowerCase() === 'all'
            ) {
              return true;
            }
            return false;
          }}
          onChange={e => {
            if (e) {
              onChange(e);
              setValue(e);
            }
          }}
        >
          <Input size="default" placeholder={placeholder} />
        </AutoComplete>
      </Form.Item>
    </>
  );
};

export default AutoCompleted;
