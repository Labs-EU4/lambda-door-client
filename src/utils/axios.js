import axios from 'axios';

axios.defaults.baseURL = `${process.env.REACT_APP_BACKEND_URL}`

export default function withAuth() {
  const token = localStorage.getItem('token');
  const instance = axios.create({
    headers: {
      'Content-Type': 'application/json',
      Authorization: `${token}`,
    },
  });

  return instance;
}
