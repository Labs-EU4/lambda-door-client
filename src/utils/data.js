export const companies = [
  {
    id: 1,
    name: 'Accenture',
    website: 'www.accenture.com.',
    location: 'Atlanta, GA',
    type: 'Business',
    logo: '',
    description: '',
  },
  {
    id: 2,
    name: 'Anthem, Inc.',
    website: 'https://www.antheminc.com',
    location: 'Atlanta, GA',
    type: 'Health',
    logo: '',
    description: '',
  },
  {
    id: 3,
    name: 'Ahalogy',
    website: 'https://www.ahalogy.com',
    location: 'Cincinnati, OH',
    type: 'Business',
    logo: '',
    description: '',
  },
  {
    id: 4,
    name: 'Bad Rabbit',
    website: 'https://www.badrabbit.com',
    location: 'Portland, OR',
    type: 'Technology',
    logo: '',
    description: '',
  },
  {
    id: 5,
    name: 'DoNotPay Inc',
    website: 'https://donotpay.com',
    location: 'Technology',
    type: 'rowValue3',
    logo: '',
    description: '',
  },
  {
    id: 6,
    name: 'Newfront Insurance',
    website: 'https://www.newfrontinsurance.com',
    location: 'San Francisco, CA',
    type: 'insurance',
    logo: '',
    description: '',
  },
  {
    id: 7,
    name: 'Unspun',
    website: 'https://www.unspuntech.com/',
    location: 'San Francisco, CA',
    type: 'Commerce',
    logo: '',
    description: '',
  },
  {
    id: 8,
    name: 'rideOS',
    website: 'https://rideos.ai',
    location: 'San Francisco, CA',
    type: 'Technology',
    logo: '',
    description: '',
  },
  {
    id: 9,
    name: 'Mesa',
    website: '',
    location: 'Austin, TX',
    type: 'Software',
    logo: '',
    description: '',
  },
  {
    id: 10,
    name: 'Paystack',
    website: 'https://paystack.com/',
    location: 'Lagos, NG',
    type: 'FinTech',
    logo: '',
    description: '',
  },
  {
    id: 11,
    name: 'BuyCoins',
    website: 'https://buycoins.africa/',
    location: 'Lagos, NG',
    type: 'Crypto',
    logo: '',
    description: '',
  },
  {
    id: 12,
    name: 'Chipper Cash',
    website: 'https://chippercash.com/',
    location: 'Lagos, NG',
    type: 'FinTech',
    logo: '',
    description: '',
  },
  {
    id: 13,
    name: 'Cowrywise',
    website: 'https://www.cowrywise.com/',
    location: 'Lagos, NG',
    type: 'FinTech',
    logo: '',
    description: '',
  },
  {
    id: 14,
    name: 'Others',
    website: '',
    location: '',
    type: '',
    logo: '',
    description: '',
  },
];
export const salaryReviews = [
  {
    id: 1,
    user_id: 1,
    company_id: 1,
    description: 'Accenture Programmer',
    salary: 95000,
    currency: 'USD',
    is_accepting_questions: 0,
  },
  {
    id: 2,
    user_id: 2,
    company_id: 3,
    description: 'Software Engineer',
    salary: 98000,
    currency: 'USD',
    is_accepting_questions: 0,
  },
  {
    id: 3,
    user_id: 3,
    company_id: 10,
    description: 'Junior Developer',
    salary: 3000000,
    currency: 'NGN',
    is_accepting_questions: 1,
  },
  {
    id: 4,
    user_id: 4,
    company_id: 1,
    description: 'Accenture Technical Specialist',
    salary: 85000,
    currency: 'USD',
    is_accepting_questions: 1,
  },
  {
    id: 5,
    user_id: 5,
    company_id: 13,
    description: 'Software Engineer',
    salary: '4800000',
    currency: 'NGN',
    is_accepting_questions: 0,
  },
];
export const interviewReviews = [
  {
    id: 1,
    user_id: 1,
    company_id: 1,
    company_name: 'Microsoft',
    text:
      'Six rounds of phone/tech interviews over a long time period. It seemed a bit scattered and could have been way more efficient. I felt like some of the interviews got repetitive. The entire process took three months and I was able to interview at another company and find a job whilst waiting to hear back.',
  },
  {
    id: 2,
    user_id: 2,
    company_id: 5,
    company_name: 'Google',
    text:
      'Very pleasant recruiters. First a phone interview that took about 15 minutes. Followed by two Skype interviews; first with a recruiter after that with a manager.Basic trouble shooting questions. They want to really get to know you as a person. Just be yourself and do not be afraid to be geeky/nerdy! Make sure you have a desk and dedicated home phone line.',
  },
  {
    id: 3,
    user_id: 4,
    company_id: 11,
    company_name: 'Amazon',
    text:
      'Completed their developer challenge and received an email offering an interview. A few preliminary questions via email. Scheduled a phone interview. The recruiter did not call me, and did not respond to my email asking why they missed the scheduled interview. I am not sure what happened, but after scheduling the interview, there is no reason to not show up.',
  },
  {
    id: 4,
    user_id: 3,
    company_id: 6,
    company_name: 'Twitter',
    text:
      'There is one phone interview for an hour. If you clear this interview, you will be invited for an onsite interview. There are five interviews onsite, all in one day. Interviews are in a casual environment. After the first two interviews, you take a break and are escorted for lunch.',
  },
  {
    id: 5,
    user_id: 5,
    company_id: 12,
    company_name: 'Slack',
    text:
      "They initially called me in for a technical interview and I didn't hear anything back for three weeks. After this I was surprised to get another call saying that I had been invited for a behavioural interview. In total the process took over two months. The interviews were both with a panel and were very thorough but the atmosphere was relaxed and friendly.",
  },

  {
    company_id: 21,
    company_name: 'Novosco',
    text:
      'I was invited to a very informal chat in the first place and shown around the office. Two weeks later I got a technical interview and I met some more people from the office. The interview lasted for an hour and had both easy and difficult questions. Two days later I got a phone call saying that I had got the job.',
  },
];

export const allInterests = [
  'Software Engineer',
  'Front End',
  'Back End',
  'Full Stack',
  'Data Science',
  'Machine Learning',
  'User Experience',
  'Mobile Development',
  'Product Manager',
];
export const jobCategories = [
  {
    id: 1,
    name: 'Software Engineer',
  },
  {
    id: 2,
    name: 'Front End',
  },
  {
    id: 3,
    name: 'Back End',
  },
  {
    id: 4,
    name: 'Full Stack',
  },
  {
    id: 5,
    name: 'Data Science',
  },
  {
    id: 6,
    name: 'Machine Learning',
  },
  {
    id: 7,
    name: 'User Experience',
  },
];
export const currencies = [
  {
    id: 1,
    name: 'USD',
    symbol: '$',
    full_name: 'USD Dollars',
  },
  {
    id: 2,
    name: 'EUR',
    symbol: '',
    full_name: 'EUROS',
  },
  {
    id: 3,
    name: 'GBP',
    symbol: '',
    full_name: 'British Pounds',
  },
  {
    id: 4,
    name: 'NAIRA',
    symbol: '',
    full_name: 'Nigerian Naira',
  },
];
