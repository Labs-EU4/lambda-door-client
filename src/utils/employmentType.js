export const employmentType = [
  {
    id: 1,
    name: 'Full-time',
  },
  {
    id: 2,
    name: 'Contract',
  },
  {
    id: 3,
    name: 'Part-time',
  },
  {
    id: 4,
    name: 'Internship',
  },
];
