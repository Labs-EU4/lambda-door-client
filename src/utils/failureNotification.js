import { notification } from 'antd';

const failureNotification = message => {
  notification.open({
    message: 'Something went wrong',
    description: message,
  });
};

export default failureNotification;
