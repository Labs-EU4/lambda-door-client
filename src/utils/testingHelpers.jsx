import React from 'react';
import { createStore, applyMiddleware, compose } from 'redux';
import { Provider } from 'react-redux';
import { Router, Route } from 'react-router-dom';
import { render } from '@testing-library/react';
import thunk from 'redux-thunk';
import { createMemoryHistory } from 'history';
import rootReducer from '../state/reducers/index';

export const TestingRouter = ({
  ComponentWithRedirection,
  RedirectUrl,
  MainRoute,
}) => (
  <>
    <Route
      path={MainRoute}
      exact={true}
      render={props => {
        return <ComponentWithRedirection {...props} />;
      }}
    />
    <Route path={RedirectUrl} render={() => <div>{RedirectUrl}</div>} />
  </>
);

export const renderWithRedux = (
  component,
  {
    initialState,
    store = createStore(
      rootReducer,
      initialState,
      compose(applyMiddleware(thunk))
    ),
  } = {},
  {
    route = '/',
    history = createMemoryHistory({ initialEntries: [route] }),
  } = {}
) => {
  return {
    ...render(
      <Provider store={store}>
        <Router history={history}>{component}</Router>
      </Provider>
    ),
    store,
    history,
  };
};
