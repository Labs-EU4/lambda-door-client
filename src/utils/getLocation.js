import axios from 'axios';
import store from '../state/store';
import { editProfile } from '../state/actions/user';

export async function showPosition(position, id) {
  const { longitude, latitude } = position.coords;

  const {
    data: { results },
  } = await axios.get(
    `https://maps.googleapis.com/maps/api/geocode/json?latlng=${latitude},${longitude}&result_type=country&key=${process.env.REACT_APP_GOOGLE_API_KEY}`
  );
  await store.dispatch(
    editProfile(
      { longitude, latitude, location: results[0].formatted_address },
      id
    )
  );
}

function showError(error) {
  switch (error.code) {
    case error.PERMISSION_DENIED:
      console.log('User denied the request for Geolocation.');
      break;
    case error.POSITION_UNAVAILABLE:
      console.log('Location information is unavailable.');
      break;
    case error.TIMEOUT:
      console.log('The request to get user location timed out.');
      break;
    case error.UNKNOWN_ERROR:
      console.log('An unknown error occurred.');
      break;
    default:
      return null;
  }
  return null;
}
export function getLocation(id) {
  if (navigator.geolocation) {
    navigator.geolocation.getCurrentPosition(
      position => showPosition(position, id),
      showError
    );
  }
}
