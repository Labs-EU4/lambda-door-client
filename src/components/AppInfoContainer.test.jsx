import React from 'react';
import * as rtl from '@testing-library/react';
import AppInfoContainer from './AppInfoContainer';
import '@testing-library/jest-dom/extend-expect';
import { renderWithRedux } from '../utils/testingHelpers';

beforeEach(rtl.cleanup);

describe('AppInfoContainer', () => {
  it('renders without crashing', () => {
    renderWithRedux(<AppInfoContainer />);
  });

  it('renders correctly', () => {
    expect(renderWithRedux(<AppInfoContainer />).baseElement).toMatchSnapshot();
  });
});
