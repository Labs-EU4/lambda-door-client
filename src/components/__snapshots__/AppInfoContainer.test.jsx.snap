// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppInfoContainer renders correctly 1`] = `
<body>
  <div>
    <div
      class="sc-bdVaJa bDDlKO"
    >
      <div
        class="sc-bwzfXH iHWQJR"
      >
        <div
          class="heading"
        >
          <h2>
            Features
          </h2>
        </div>
        <div
          class="sc-htpNat iFaotm"
        >
          <p
            class="left-outer"
          >
            <i
              aria-label="icon: slack-circle"
              class="anticon anticon-slack-circle icon"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="slack-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zM361.5 580.2c0 27.8-22.5 50.4-50.3 50.4a50.35 50.35 0 0 1-50.3-50.4c0-27.8 22.5-50.4 50.3-50.4h50.3v50.4zm134 134.4c0 27.8-22.5 50.4-50.3 50.4-27.8 0-50.3-22.6-50.3-50.4V580.2c0-27.8 22.5-50.4 50.3-50.4a50.35 50.35 0 0 1 50.3 50.4v134.4zm-50.2-218.4h-134c-27.8 0-50.3-22.6-50.3-50.4 0-27.8 22.5-50.4 50.3-50.4h134c27.8 0 50.3 22.6 50.3 50.4-.1 27.9-22.6 50.4-50.3 50.4zm0-134.4c-13.3 0-26.1-5.3-35.6-14.8S395 324.8 395 311.4c0-27.8 22.5-50.4 50.3-50.4 27.8 0 50.3 22.6 50.3 50.4v50.4h-50.3zm83.7-50.4c0-27.8 22.5-50.4 50.3-50.4 27.8 0 50.3 22.6 50.3 50.4v134.4c0 27.8-22.5 50.4-50.3 50.4-27.8 0-50.3-22.6-50.3-50.4V311.4zM579.3 765c-27.8 0-50.3-22.6-50.3-50.4v-50.4h50.3c27.8 0 50.3 22.6 50.3 50.4 0 27.8-22.5 50.4-50.3 50.4zm134-134.4h-134c-13.3 0-26.1-5.3-35.6-14.8S529 593.6 529 580.2c0-27.8 22.5-50.4 50.3-50.4h134c27.8 0 50.3 22.6 50.3 50.4 0 27.8-22.5 50.4-50.3 50.4zm0-134.4H663v-50.4c0-27.8 22.5-50.4 50.3-50.4s50.3 22.6 50.3 50.4c0 27.8-22.5 50.4-50.3 50.4z"
                />
              </svg>
            </i>
            No need to sign up: Lambda students have automatic access. Just sign in via Slack.
          </p>
          <p
            class="left-center"
          >
            <i
              aria-label="icon: edit"
              class="anticon anticon-edit icon"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="edit"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M880 836H144c-17.7 0-32 14.3-32 32v36c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-36c0-17.7-14.3-32-32-32zm-622.3-84c2 0 4-.2 6-.5L431.9 722c2-.4 3.9-1.3 5.3-2.8l423.9-423.9a9.96 9.96 0 0 0 0-14.1L694.9 114.9c-1.9-1.9-4.4-2.9-7.1-2.9s-5.2 1-7.1 2.9L256.8 538.8c-1.5 1.5-2.4 3.3-2.8 5.3l-29.5 168.2a33.5 33.5 0 0 0 9.4 29.8c6.6 6.4 14.9 9.9 23.8 9.9z"
                />
              </svg>
            </i>
            Leave company reviews, salary reviews and interview reviews, publicly or anonymously.
          </p>
          <p
            class="right-center"
          >
            <i
              aria-label="icon: info-circle"
              class="anticon anticon-info-circle icon"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="info-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm32 664c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8V456c0-4.4 3.6-8 8-8h48c4.4 0 8 3.6 8 8v272zm-32-344a48.01 48.01 0 0 1 0-96 48.01 48.01 0 0 1 0 96z"
                />
              </svg>
            </i>
            Access up to date information about companies in your area or around the world.
          </p>
          <p
            class="right-outer"
          >
            <i
              aria-label="icon: mail"
              class="anticon anticon-mail icon"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="mail"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M928 160H96c-17.7 0-32 14.3-32 32v640c0 17.7 14.3 32 32 32h832c17.7 0 32-14.3 32-32V192c0-17.7-14.3-32-32-32zm-80.8 108.9L531.7 514.4c-7.8 6.1-18.7 6.1-26.5 0L189.6 268.9A7.2 7.2 0 0 1 194 256h648.8a7.2 7.2 0 0 1 4.4 12.9z"
                />
              </svg>
            </i>
            Email referrals: ask questions at the click of a button about the companies you're interested in.
          </p>
        </div>
        <a
          class="back-link"
        >
          Go back Â 
          <i
            aria-label="icon: up-circle"
            class="anticon anticon-up-circle"
            style="font-size: 22px; color: rgb(187, 19, 51);"
          >
            <svg
              aria-hidden="true"
              class=""
              data-icon="up-circle"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm178 555h-46.9c-10.2 0-19.9-4.9-25.9-13.2L512 460.4 406.8 605.8c-6 8.3-15.6 13.2-25.9 13.2H334c-6.5 0-10.3-7.4-6.5-12.7l178-246c3.2-4.4 9.7-4.4 12.9 0l178 246c3.9 5.3.1 12.7-6.4 12.7z"
              />
            </svg>
          </i>
        </a>
      </div>
    </div>
  </div>
</body>
`;
